db/schema.prisma

```diff
@@ -63,3 +63,11 @@ model Report {
   createdAt   DateTime @default(now())
   updatedAt   DateTime @updatedAt
 }
+
+model Emblem {
+  id          String    @id @default(uuid())
+  name        String?
```

Comment:
nome é opcional?

Reply:
acabei confundindo o uso do "?", é pra not null

Reply:
Precisa de algum ajuste?